### PR TITLE
added_user_contact_phone

### PR DESCRIPTION
**Pull request by AliakseiRabushka**

### CODE REVIEW

File: src/main/resources/db/migration/V2022_08_19_1__added_user_contact_phone.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (Sveta130) [inline src/main/resources/db/migration/V2022_08_19_1__added_user_contact_phone.sql:1]: why this script is empty?

File: src/main/java/com/example/kindergarten/converter/UserConverter.java [hunk 2]
```diff
@@ -17,6 +17,7 @@ public static Optional<UserEntity> toEntity(UserDto userDto) {
             .role(userDto.getRole())
             .accountNumber(userDto.getAccountNumber())
             .address(userDto.getAddress())
+                .contactPhone(userDto.getContactPhone())
```
Comment (belnarto) [inline src/main/java/com/example/kindergarten/converter/UserConverter.java:22]: fix code format here please

File: src/main/resources/db/migration/V2022_08_19_1__added_user_contact_phone.sql (hunk 1 above)
Comment (AliakseiRabushka) [reply src/main/resources/db/migration/V2022_08_19_1__added_user_contact_phone.sql:1]: this script is fixed.

**Review by AliakseiRabushka — COMMENTED** [review]
added user contact phone fixed

**Review by AliakseiRabushka — COMMENTED** [review]
fixed script

File: src/main/java/com/example/kindergarten/converter/UserConverter.java (hunk 2 above)
Comment (belnarto) [reply src/main/java/com/example/kindergarten/converter/UserConverter.java:22]: тут не подфиксал

**Review by belnarto — CHANGES_REQUESTED** [review]

**Review by AliakseiRabushka — COMMENTED** [review]
user converter fixed

**Review by belnarto — APPROVED** [review]

**Pull request merged**